test/containers/ExampleGridItem.js

```diff
@@ -48,6 +48,7 @@ export default class Item extends Component {
             <button
               id={`add-more-${itemIdx}`}
               onClick={() => fetchMore().then(addRelatedItems)}
+              type="submit"
```

Comment:
@pfarejowicz same here - change to `type="button"`

Reply:
Done, same comment as above but this is a test component so Im not too concerned